rockcraft-pack/action.yml

```diff
@@ -0,0 +1,26 @@
+name: "Build a ROCK"
+description: "Pack an OCI image with Rockcraft"
+inputs:
+  path:
+    description: >
+      The location of the Rockcraft project. 
+      
+      Defaults to the base of the repository.
+    default: '.'
+  rockcraft-args:
```

Comment:
can we start without this and perhaps a verbosity flag that takes an enum?

Reply:
done